src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.CodeAction_Property.cs

```diff
@@ -27,28 +27,53 @@ internal partial class ImplementInterfaceCodeAction
             {
                 var factory = this.Document.GetLanguageService<SyntaxGenerator>();
                 var comAliasNameAttribute = compilation.ComAliasNameAttributeType();
+                var tupleElementNamesAttribute = compilation.TupleElementNamesAttributeType();
 
-                var getAccessor = property.GetMethod == null
-                    ? null
-                    : CodeGenerationSymbolFactory.CreateAccessorSymbol(
-                        property.GetMethod.RemoveInaccessibleAttributesAndAttributesOfType(
-                            accessibleWithin: this.State.ClassOrStructType,
-                            removeAttributeType: comAliasNameAttribute),
-                        attributes: null,
-                        accessibility: accessibility,
-                        explicitInterfaceSymbol: useExplicitInterfaceSymbol ? property.GetMethod : null,
-                        statements: GetGetAccessorStatements(compilation, property, generateAbstractly, cancellationToken));
-
-                var setAccessor = property.SetMethod == null
-                    ? null
-                    : CodeGenerationSymbolFactory.CreateAccessorSymbol(
-                        property.SetMethod.RemoveInaccessibleAttributesAndAttributesOfType(
-                            accessibleWithin: this.State.ClassOrStructType,
-                            removeAttributeType: comAliasNameAttribute),
-                        attributes: null,
-                        accessibility: accessibility,
-                        explicitInterfaceSymbol: useExplicitInterfaceSymbol ? property.SetMethod : null,
-                        statements: GetSetAccessorStatements(compilation, property, generateAbstractly, cancellationToken));
+                IMethodSymbol getAccessor;
+                if (property.GetMethod == null)
+                {
+                    getAccessor = null;
```

Comment:
in the helper you can just return null.  We're not a fan of multiple paths of initialization in IDE code.